### DISCUSSION TITLE
How can I disable the conversion from markdown to HTML?

### DISCUSSION THREAD

**Discussion by gianarb** [yuin/goldmark#168] [General]
Hello! I am sure I am doing something unuseful, so let me know if this is not supported but in practice, I am not using this library to generate HTML but I am using it to parse markdown and to generate a YAML manifest that another program will use.

The YAML manifest has a field where I would like to write the body of the Markdown file but not converted to HTML, in its original markdown format.

Right now I am using:

```golang
	md := goldmark.New(
		goldmark.WithExtensions(
			meta.Meta,
		),
	)
	readme, err := ioutil.ReadAll(file)
	if err != nil {
		return errors.Wrap(err, "error reading the README.md")
	}
	var buf bytes.Buffer
	context := parser.NewContext()
	if err := md.Convert(readme, &buf, parser.WithContext(context)); err != nil {
		return errors.Wrap(err, "error converting the README.md proposal")
	}

```

But reading `buf` I get back HTML, I would like to get the original markdown instead.

Is it possible?
Thanks a lot

**Comment by moorereason** [comment]
Don't you already have that in `readme`?

**Reply by gianarb** [reply]
yes but readme contains the all content of the file.

```md
---
mata: data
---

content
```

I want to parse the metadata because I use them elsewhere, but I don't want to convert the content to html

**Reply by bainss** [reply]
I have a similar requirement to parse the metadata in MD, 

`
---
title: '[SB] KB - Operator - Operational Guide'
tags:
  - operator
  - kb
  - guide
---
`
but not render it as HTML, use in a custom renderer

**Comment by Elara6331** [comment]
Using a markdown parser/renderer just to extract metadata seems overkill to me. Something like this should suffice: https://go.dev/play/p/8_SQf1eV2aO.
